vector/src/main/java/im/vector/app/features/notifications/NotificationDrawerManager.kt

```diff
@@ -123,6 +124,22 @@ class NotificationDrawerManager @Inject constructor(
         }
     }
 
+    /**
+     * Should be called when the application is currently opened and showing timeline for the given threadId.
+     * Used to ignore events related to that thread (no need to display notification) and clean any existing notification on this room.
+     */
+    fun setCurrentThread(threadId: String?) {
```

Comment:
Maybe merge with the method `setCurrentRoom`, just adding the parameter `threadId: String?` to it?

Reply:
OK